cmd/generate.go

```diff
@@ -65,13 +57,13 @@ func newGenerateCmd() *cobra.Command {
 	}
 
 	f := generateCmd.Flags()
-	f.StringVarP(&gc.apimodelPath, "api-model", "m", "", "path to the apimodel file")
-	f.StringVarP(&gc.outputDirectory, "output-directory", "o", "", "output directory (derived from FQDN if absent)")
-	f.StringVar(&gc.caCertificatePath, "ca-certificate-path", "", "path to the CA certificate to use for Kubernetes PKI assets")
-	f.StringVar(&gc.caPrivateKeyPath, "ca-private-key-path", "", "path to the CA private key to use for Kubernetes PKI assets")
-	f.StringArrayVar(&gc.set, "set", []string{}, "set values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)")
-	f.BoolVar(&gc.noPrettyPrint, "no-pretty-print", false, "skip pretty printing the output")
-	f.BoolVar(&gc.parametersOnly, "parameters-only", false, "only output parameters files")
+	f.StringVarP(&currentConfig.CLIConfig.Generate.APIModel, "api-model", "m", defaultConfigValues.CLIConfig.Generate.APIModel, "path to the apimodel file")
```

Comment:
can we assign this to a variable `&currentConfig.CLIConfig.Generate` and reuse that ?

We can follow this same pattern in other places as well :)

Reply:
done